later/lib/domain/remap/record_remapper_impl.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:later/database/models/record_model.dart';
+import 'package:later/domain/models/record_model.dart';
+import 'package:later/domain/remap/record_remapper.dart';
+
+class RecordRemapperImpl extends RecordRemapper {
```

Comment:
We use the `injectable` package for dependency injection (please have a look at it if you have not yet). Hence we must annotate this class with `@LazySingleton(as: RecordRemapper)` so injectable will generate the needed code for us.